docs/docs/user/tools/capella/troubleshooting.md

```diff
@@ -0,0 +1,45 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Capella Troubleshooting
+
+!!! question "My Capella crashed. What should I do?"
+
+    Capella can crash for many different reasons. Many issues are issues with
+    Eclipse Capella itself, so we try to escalate issues there. Bugs can be
+    reported to the Eclipse Capella team directly in the
+    [Github repository](https://github.com/eclipse/capella/issues).
+
+    Administrators can see the logs of all sessions to identify the issues
+    remotely. In addition, the session owner can also see the events in the UI.
+    In your session, please follow these steps:
+
+    1. Open the Capella search
+    1. Search for `Event logs` and click on the following entry:
+    1. You should now see the list of events and you can try to find an entry with
+       a matching timestamp.
```

Comment:
When I started a new capella session and searched for `Event log` in the capella search, no results were displayed. So is it only visible if there was at least one crash/event or should I find it in any case?

Also, I seem to be missing an image in the second item as it says *"[...] and click on the following entry: "* without further information after the colon.

Reply:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/7419ce91-82c4-4ff9-a593-08bd3d23660d)

I can find Event log, can you search again?

Reply:
Second comment is valid and fixed.